acestep/gradio_ui/interfaces/result.py

```
@@ -6,6 +6,93 @@
+        "codes_display": codes_display,
+        "convert_to_codes_btn": convert_to_codes_btn,
+        "score_display": score_display,
+        "score_btn": score_btn,
+        "lrc_display": lrc_display,
+        "lrc_btn": lrc_btn,
+        "save_lrc_btn": save_lrc_btn,
+        "lrc_download_file": lrc_download_file,
+    }
+
+
 def create_results_section(dit_handler) -> dict:
```

Ruff (0.15.0)

[warning] Unused function argument: `dit_handler`

(ARG001)